examples/archive.html

```diff
@@ -95,7 +95,9 @@ <h3 id="offcanvasRightLabel">Images</h3>
     form.addEventListener('submit', (event) => {
       event.preventDefault();
       const url = input.value.replace('details', 'metadata');
-      let fetchedFrom
+      let splitUrl = url.split('/');
```

Comment:
Hi, it looks like you may be mixing code from separate PRs here. This can happen if you work from your `main` branch and don't separate work into different branches.  Could you reset it so these unrelated changes are not included in the PR? 

You could try this -- https://stackoverflow.com/questions/7147270/hard-reset-of-a-single-file or just copy the version from the publiclab main branch (https://github.com/publiclab/Leaflet.DistortableImage) over your own and add it so there are no changes shown here. Let me know if you need help!